### PR TITLE
fix for #9 & #12

### PR DESCRIPTION
**Pull request by k0shk0sh**

### CODE REVIEW

File: shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java [hunk 1]
```diff
@@ -86,6 +93,17 @@ protected void dispatchDraw(Canvas canvas) {
         }
     }
 
+    @Override public void setVisibility(int visibility) {
```
Comment (veghtomi) [inline shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java]: @override should be in a different line like in other cases.

File: shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java [hunk 2]
```diff
@@ -278,4 +296,12 @@ private int getColor(int id) {
             return getResources().getColor(id);
         }
     }
+
+    private void removeGlobalListener(ViewTreeObserver.OnGlobalLayoutListener listener) {
```
Comment (veghtomi) [inline shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java]: Call this method removeGlobalLayoutListener(...)

**Review by veghtomi — CHANGES_REQUESTED** [review]

File: shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java (hunk 1 above)
Comment (k0shk0sh) [reply shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java]: Ooops, that slipped by my own style 😕

**Comment by k0shk0sh** [comment]
@veghtomi please review & update accordingly.

**Review by veghtomi — APPROVED** [review]

**Pull request merged**